.env.example

```diff
@@ -12,3 +12,5 @@ REACT_APP_ENABLE_NO_METAMASK_PROMPT='true'
 REACT_APP_STEP_MODAL_DONE_STATUS_VISIBILITY_TIME='3500'
 REACT_APP_COLLECTIBLES_SOURCE='opensea'
 REACT_APP_COLLECTIBLE_NAME='CryptoKitties'
+REACT_APP_ERC20_THEME_NAME = 'WHITE_THEME'
+REACT_APP_ERC721_THEME_NAME = 'DARK_THEME'
```

Comment:
Req: I believe this should be `WHITE`, and the above line should be `DARK`

Reply:
Yes, thanks for notice that, I'll make the changes